csv-bulk-signature/src/main.ts

```diff
@@ -138,16 +131,17 @@ async function checkHeadersToBeSame(contact: Contact) {
 }
 
 async function zipUpFile() {
-  const folderPath = "./dist/signatures";
-  const files = fs.readdirSync(folderPath);
-  const filesHtm = files.filter(
+  const signatureFolder = fs.readdirSync(SIGNATURES_PATH);
+  const files = signatureFolder.filter(
```

Comment:
- use const dist path
- rename `files` to the folder we're targeting
- rename `filesHtm` since we including the report too